docs/src/index.md

```diff
@@ -4,26 +4,38 @@
+`PowerSystems.jl` is a [`Julia`](http://www.julialang.org) package Power Systems Modeling that provides a rigorous data model using Julia structures to enable power systems modeling.
 
-The storage of power system data has traditionally been done with general use tables based on the power flow problem. However, tables are not inherently designed to store data with mixed data representations or hierarchal structures. This limitation was addressed in the early 1990's with the advent of automation, spurred by increasingly complex data needs for power systems operations. The industry required standardized models to exchange larger and more complex sets of information, and so resorted to an object-oriented data model. The CIM was developed and later made a standard maintained by the IEC - Technical Committee 57 Working Group 13. The aim was to provide a standard definition for power system components that could be used in automated EMS and asset-management databases.
+`PowerSystems.jl` provides tools to prepares and processes data useful for electric energy systems modelling. This package serves two purposes:
 
-From the computer engineering point-of-view, using tables would require to re-write a lot of code. In most languages with strongly type hierarchies, using a tree structure it is advantageous. This is particularly relevant for code re-use and data encapsulation. Collecting all potential devices is not possible; neither is it desirable. However, it is necessary to have the capability to extend the data model such that different user can store their data in an organized ontology.
+1. It facilitates the development and open sharing of custom datasets between users and computer systems.
+2. It provides a data model that imposes discipline on model specification, addressing the challenge of design and terminology choices when sharing code and data.
 
-For instance, take the data hierarchy for thermal generation devices shown in Fig. \ref{fig:PS_thermal}. Generation is classified by the distinctive operational characteristics of the generators and then further categorized by the data required to model a specific technology.
+The main features include:
 
-For more detailed documentation of each object in the library, see the API/[PowerSystems](@ref) page.
+- Extensive library of data structures for power systems modeling.
+- Parsing capabilities from common data formats (PSS/e raw and dyr, Matpower and CSV)
 
-## Installation
+`PowerSystems.jl` documentation and code are organized according to the needs of different users depending on their skillset and requirements. In broad terms there are three categories:
 
-The latest stable release of PowerModels can be installed using the Julia package manager with
+- **Modeler**: Users that want to use `PowerSystems.jl` to develop a new data set for a particular analysis or experiment.
```

Comment:
Would it be better to say `Users that want to run a particular analysis or experiment.`?